scalarUDF.go

```diff
@@ -1,12 +1,11 @@
 package duckdb
```

Comment:
please rename to `scalar_udf.go`

Reply:
Done https://github.com/marcboeker/go-duckdb/pull/380